packages/gatsby/src/utils/get-latest-apis.ts

```diff
@@ -1,20 +1,24 @@
-const path = require(`path`)
-const fs = require(`fs-extra`)
-const axios = require(`axios`)
+import path from "path"
+import fs from "fs-extra"
+import axios from "axios"
 
 const API_FILE = `https://unpkg.com/gatsby/apis.json`
 const ROOT = path.join(__dirname, `..`, `..`)
 const OUTPUT_FILE = path.join(ROOT, `latest-apis.json`)
 
-module.exports = async function getLatestAPI() {
+export interface IAPIResponse {
+  [key: string]: any
+}
+
+export const getLatestAPIs = async (): Promise<IAPIResponse> => {
```

Comment:
We need to define the interface of API response but this was too difficult for me. plz help 🙏

Reply:
it looks like it's some object with `{ browser: {}, node: {}, ssr: {} }`, I would probably just add a console.log to `data` here and run the code so you can see what it prints out. 

I think this gets triggered by having a gatsby plugin with exports that are not part of the gatsby spec.

Reply:
It has so many properties but I determined below for now.
```ts
interface IAPIResponse {
  ssr: {
    [key: string]: object
  }
  browser: {
    [key: string]: object
  }
  node: {
    [key: string]: object
  }
}
```

Reply:
i'd prefer to be explicit. Let's properly type it please!